tox.ini

```diff
@@ -19,11 +19,15 @@ commands =
     py.test {posargs}
 
 [testenv:docs]
+changedir = {toxinidir}/docs
+commands =
+    sphinx-build -b html -d {envtmpdir}/doctrees .  {envtmpdir}/html
+
+[testenv:docs-lint]
 deps = -r{toxinidir}/requirements/lint.txt
```

Comment:
Here. You just need `rstcheck` instead all the linting packages, I'd say

(or maybe creating another file `docs-lint.txt`?)

Reply:
We need also sphinx for more integration https://github.com/myint/rstcheck#sphinx

Reply:
Nice! So, I'd say that's better to have a separate reqs file.